docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :support:`255` Add support for provisioning mass based resources
```

Comment:
nit: I'd argue this could be a `feature`

Reply:
Makes sense. Modified to:
```
* :feature:`255` Add support for provisioning of resources by mass
```